docs/preprocessing.rst

```diff
@@ -276,6 +276,53 @@ Volumetric output spaces include ``T1w`` (default) and ``MNI152NLin2009cAsym``.
   signal model divided by the variance of the error of the signal model.
 
 
+Transforms
+^^^^^^^^^^
+
+QSIPrep will write out a series of transforms needed to map between the different spaces.
+
+.. code-block::
+
+  sub-<label>/
+    anat/
+      sub-<label>_from-anat_to-ACPC_mode-image_xfm.mat
+      sub-<label>_from-ACPC_to-anat_mode-image_xfm.mat
+      sub-<label>_from-ACPC_to-<output-space>_mode-image_xfm.h5
+      sub-<label>_from-<output-space>_to-ACPC_mode-image_xfm.h5
+    ses-<label>/
+      anat/
+        sub-<label>_ses-<label>_from-orig_to-anat_mode-image_xfm.txt
+        sub-<label>_ses-<label>_from-anat_to-orig_mode-image_xfm.txt
+
+
+.. important::
+
+  Motion correction, eddy current correction, and distortion correction are performed with EDDY,
```

Comment:
```suggestion
  Motion correction, eddy current correction, and distortion correction are performed with Eddy,
```

Reply:
Not sure how they picked this capitalization, but I think this is the official version